android/app/src/main/java/io/flutter/app/FlutterMultiDexApplication.java

```diff
@@ -0,0 +1,20 @@
+// Generated file.
+// If you wish to remove Flutter's multidex support, delete this entire file.
+
+package io.flutter.app;
+
+import android.content.Context;
+import androidx.annotation.CallSuper;
+import androidx.multidex.MultiDex;
```

Comment:
Not sure that we need Multidex for such small app.

Reply:
by default in gradle minSdkVersion flutter.minSdkVersion == 19. The plugin yandex_mapkit requires a higher Android SDK version. and I change it to 21. And this actions bring to install Multidex. When i try to delete it, or modify this file i get error.

Reply:
and now i enter comand flutter clean, delete this file and run app. in build was no quastion about install multidex. i am perplexe